crates/bevy_reflect/src/list.rs

```diff
@@ -522,4 +523,24 @@ mod tests {
             assert_eq!(index, value);
         }
     }
+    // This test needs to be run in release mode
+    #[test]
```

Comment:
This doesn't seem to be marked ignore :)

Reply:
i must have missed to re-commit